<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>244</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>518</width>
    <height>368</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 44, 54);
color:white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>350</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>FreeSerif</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(39, 44, 54);
color:white;</string>
     </property>
     <property name="title">
      <string>PARAMETRES DE ROTATION</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>43</y>
        <width>481</width>
        <height>71</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>479</width>
         <height>69</height>
        </rect>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>461</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="labelAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white ;</string>
           </property>
           <property name="text">
            <string>ANGLE DE ROTATION :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit">
           <property name="styleSheet">
            <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: white;
color:black ;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>102</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton
{
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white 
}
QPushButton:pressed{ background-color: blue; }</string>
         </property>
         <property name="text">
          <string>VALIDER</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton
{
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white 
}
QPushButton:pressed{ background-color: blue; }</string>
         </property>
         <property name="text">
          <string>ANULER</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
